package Current_Exception is

  function Exception_Name return String;

end Current_Exception;

